#Change camera stream
string new_camera
---
